linux_os/guide/system/logging/journald/socket_systemd-journal-remote_disabled/bash/shared.sh

```diff
@@ -0,0 +1,4 @@
+# platform = multi_platform_all
+
+{{{ bash_service_command("disable", "systemd-journal-remote.socket") }}}
```

Comment:
This really only disables the service. But we want it masked. I think you will not be able to use existing macro.
you can either create a new macro for masking of something or you can just not use macro.
I think the best inspiration is here:
https://github.com/ComplianceAsCode/content/blob/7a0f2e0db6eda8d3a153c71779d8e636028d55d8/shared/templates/service_disabled/bash.template#L9